config.js

```diff
@@ -81,7 +81,10 @@ module.exports = {
     //            the network.
     onUncaughtException: 'ignore',
 
-    // Allows to ignore the requests X-Cloud-Trace-Context -header if set
+    // EXPERIMENTAL:
+    // Allows to ignore the requests X-Cloud-Trace-Context -header if set. This will
```

Comment:
s/This will/Setting this to true will/.

Why would one want this behaviour? 

Doesn't setting this to true also imply that a local sampling decision is made an the sampling/tracing decision from the incoming request is ignored?